packages/mds-policy-service/service/validators.ts

```diff
@@ -107,7 +107,7 @@ const micromobilityStateMap = MICRO_MOBILITY_VEHICLE_STATES.reduce<{
     nullable: true
   }
   return acc
-}, {}) as { [S in MICRO_MOBILITY_VEHICLE_STATE]: MICRO_MOBILITY_VEHICLE_EVENT_ARRAY_ENUM }
```

Comment:
What's this change about?

Reply:
It used to be needed, at first, to that that `[S in MICRO_MOBILITY_VEHICLE_STATE]:` key was always-present, but the base-type of `[S in MICRO_MOBILITY_VEHICLE_STATE]?:` (optional) turned out to be okay, after working through the final schema.